config.py

```diff
@@ -1,6 +1,23 @@
 import logging
 from app.vcap_services import get_database_uri
+from app import app
```

Comment:
Not needed.
Refer to comment in line 11.

Reply:
addressed